static/js/containers/ProfilePage_test.js

```diff
@@ -66,10 +66,7 @@ describe("ProfilePage", function() {
       );
 
       renderComponent(`/profile/${page}`).then(([component, div]) => {  // eslint-disable-line no-unused-vars
```

Comment:
same comment here as above, mind ignoring `component` so we can remove the rule disable?